azure-mgmt-containerinstance/azure/mgmt/containerinstance/version.py

```diff
@@ -0,0 +1,13 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+VERSION = "2017-08-01-preview"
+
```

Comment:
That's a good point, I should use package-version on my side (should not be on the readme.dm)